docs/data/destinations/planhat-event-streaming.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Planhat Event Streaming
+description: Stream Amplitude events to Planhat.
+status: new
+---
+
+!!!beta "This feature is in open beta"
+
+    This feature is in open beta and is in active development. Contact the [Planhat support team]([https://userlist.com/](https://www.google.com/url?q=https://support.planhat.com/en/&sa=D&source=docs&ust=1681362728518389&usg=AOvVaw0HZBKn1FHVmfkZ7_tigFKN)) for support with this integration.
```

Comment:
probably get rid of `userlist` info here?

Reply:
Fixed it.